tools/generators/files_and_groups/src/Generator/Elements/Elements+MainGroup.swift

```diff
@@ -0,0 +1,26 @@
+import PBXProj
+
+extension Generator.Elements {
+    /// Creates the main `PBXGroup`.
+    static func mainGroup(
+        // FIXME: Take pre-sorted identifiers instead
```

Comment:
Maybe it's a local thing but GH is rendering these lines in a weird way for me:
![image](https://github.com/MobileNativeFoundation/rules_xcodeproj/assets/10197663/a35648b6-2972-470a-b5b1-88892512e87f)

Reply:
It's just getting confused by the comment above it.